README.md

```diff
@@ -37,6 +37,26 @@ Additionally when a section `sharding_key` is not empty in YAML schema call of
 function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
 fields: `space_name` with type `string` and `sharding_key` with type `array`.
 
+Similarly for `sharding_func`: when a section `sharding_func` is not empty in YAML schema call of
```

Comment:
AFAIR current version of code creates a space even user didn't specify `sharding_func` section in a schema. And I think it's a preferred behavior.

Reply:
Updated

Note: I guess same changes are fair for `_ddl_sharding_key` space info too

```
-Additionally when a section `sharding_key` is not empty in YAML schema call of
-function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
+Call of function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
 fields: `space_name` with type `string` and `sharding_key` with type `array`.
 
-Similarly for `sharding_func`: when a section `sharding_func` is not empty in YAML schema call of
-function `ddl.set_schema(schema)` creates a space `_ddl_sharding_func` with three
-fields: `space_name` with type `string`, `sharding_func_name` with type `string` and
-`sharding_func_body` with type `string`.
+Similarly for `sharding_func`: call of function `ddl.set_schema(schema)` creates
+a space `_ddl_sharding_func` with three fields: `space_name` with type `string`,
+`sharding_func_name` with type `string` and `sharding_func_body` with type `string`.
```

Reply:
> Note: I guess same changes are fair for _ddl_sharding_key space info too

it is better to make a separate patch with this change

Reply:
Add in #78